src/js/tracks/track-list.js

```diff
@@ -115,7 +116,8 @@ class TrackList extends EventTarget {
      */
     this.trigger({
       track,
-      type: 'removetrack'
+      type: 'removetrack',
+      target: this
```

Comment:
I think the target here is intended to be the tracklist that the event is triggered on and not document.